mongo/tox.ini

```
@@ -2,7 +2,7 @@
 minversion = 2.0
 basepython = py38
 envlist =
-    py{27,38}-{3.2.10,3.4,3.5,4.1}
+    py{27,38}-{3.2.10,3.4,3.5,4.1,4.2}
```

```suggestion
    py{27,38}-{3.5,4.1,4.2}
```
3.2 EOLed in September, 2018
3.4 EOLed in January 2020